File: android/src/main/kotlin/uz/udevs/udevs_video_player/models/PremierStreamResponse.kt
package uz.udevs.udevs_video_player.models

data class PremierStreamResponse(
    val file_info: List<FileInfo?>?
) {
    data class FileInfo(
        val duration: Int?,
        val file_name: String?,
        val height: Int?,
        val quality: String?,
        val width: Int?
    )
}

File: ios/Classes/Player/Models/PremierStreamResponse.swift
//
//  PremierStreamResponse.swift
//  udevs_video_player
//
//  Created by Udevs on 07/10/22.
//

import Foundation

struct PremierStreamResponse: Decodable {
    var file_info : [FileInfo]?
}

struct FileInfo: Decodable{
    var quality: String?
    var file_name: String?
    var duration: Int?
    var width: Int?
    var height: Int?
}
